argo_qa/solrconfig.xml

```diff
@@ -98,50 +100,32 @@
         topic_tesim^10
 
         collection_title_tesim^5
-        source_id_ssim^5
-
-        exploded_project_tag_ssim^10
-        exploded_nonproject_tag_ssim^5
-        exploded_tag_ssim^5
-        tag_ssim^5
 
         objectId_tesim^5
-        identifier_ssim^5
```

Comment:
it makes no sense to boost phrase and token-n-gram matches for string (single token) fields.  These changes are all part of the as yet unused experimental requestHandler.